### DISCUSSION TITLE
[Desktop] Cannot load svg starting from "Uno.Sdk": "5.2.83"

### DISCUSSION THREAD

**Discussion by kucint** [unoplatform/uno#16456] [Q&A] [answered]
Staring form "Uno.Sdk": "5.2.83" the following code does not work anymore:
```
public MainPage() => this
    .Background(ThemeResource.Get<Brush>("ApplicationPageBackgroundThemeBrush"))
    .Content(
        new Image()
        .Source("Controls/Assets/Icons/splash_screen.svg")
        .Width(450)
        .Height(450));
```
Neither not working on latest uno sdks:
- "5.2.92"
- "5.3.0-dev.708" 

How to reproduce:
- create a brand new uno "desktop" project
- modify main page as above
- add `splash_screen.svg` to "Controls/Assets/Icons" folder
- set build action to Content for `splash_screen.svg` file

Result:
- svg does not display at all.
- in host window you may see an error message:
![image](https://github.com/unoplatform/uno/assets/54866088/d7bb073b-1447-4f64-a40e-92164ecabd0e)
-  the [Uno Platform Features](https://aka.platform.uno/singleproject-features) claims that 
"Svg option is not needed for WebAssembly, Desktop, and WinAppSDK." so I didn't include it.
- After adding Svg option to the project
```
    <UnoFeatures>
      CSharpMarkup;
      Hosting;
      Logging;
      MVUX;
      Configuration;
      ExtensionsCore;
      Svg;
    </UnoFeatures>
```
  the error message above is no longer reported, but svg is still not visible. 

Tested on both TargetFrameworks on Windows 11, VS:17.9.6 :
- net8.0-desktop;
- net8.0-windows10.0.19041;

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report! Multiple things are happening here.

1. The documentation is incorrect about not needed the `Svg` feature for desktop. We'll adjust this.
2. Using an [Svg file](https://platform.uno/docs/articles/features/svg.html) is subject to the manipulations made by [`Uno.Resizetizer`](https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html), which means that to access the image, you'll need to use the `.png` extension instead. If you really want to use an `svg` file, you'll need to place the file inside an `/Svg/` folder so it does not get picked by by resizetizer.
3. C# Markup seems to have an issue with the handling of relative paths, we'll track it in https://github.com/unoplatform/uno/issues/16458. In the meantime, use a path like this `ms-appx:///Controls/Assets/Icons/splash_screen.svg`.

**Reply by kucint** [reply]
I was not aware of "[Svg file](https://platform.uno/docs/articles/features/svg.html) is subject to the manipulations made by [Uno.Resizetizer](https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html)" and related performance issues. 
Thanks @jeromelaban for info!
and yes, the workaround fixes the issue :)

**Reply by limeniye** [reply]
@jeromelaban, in previous version of Uno.Resizetizer I had a good quality pictures via **.svg** format for WinUI.
After the packages update I had to change the **.svg** to **.png** (because the **.svg** format doesn't work now for me)
When I have changed the formats to .png -- the quality of pictures were downgraded.

I have tried to move the image file to the ```/Svg/``` folder
```ms-appx:///ProjectName/Assets/Svg/somefile.svg```
and 
```ms-appx:///ProjectName/Svg/somefile.svg```
 but it didn't display.

**Reply by jeromelaban** [reply]
@limeniye it's best to open an issue instead, thanks!

**Reply by limeniye** [reply]
@jeromelaban https://github.com/unoplatform/uno/issues/16506
